cache_tables/handler/scans_info.py

```diff
@@ -75,13 +76,13 @@ def get_odin_data(
     max_retries: int = MAX_RETRIES,
 ) -> dict:
     url = f"{api_base}/{endpoint}"
-    response = requests.get(url, timeout=365)
     retries = max_retries
     while retries > 0:
         try:
+            response = requests.get(url, timeout=365)
             response.raise_for_status()
             break
-        except requests.exceptions.HTTPError as msg:
+        except (HTTPError, RequestException) as msg:
             retries -= 1
             if retries == 0:
                 raise RetriesExhaustedError(
```

Comment:
I'm not sure, isn't it better to stop the task execution and let the task retry a couple of minutes (hours?) later?

Reply:
Fixed in two new commits.